docker-compose.yaml

```diff
@@ -272,8 +272,8 @@ services:
       - INGEST_LOG_LEVEL=DEFAULT
       - INGEST_EDGE_BUFFER_SIZE=64
       - INGEST_DYNAMIC_MEMORY_THRESHOLD=0.80
-      - INGEST_DISABLE_DYNAMIC_SCALING=${INGEST_DISABLE_DYNAMIC_SCALING:-false}
-      #- INGEST_DISABLE_DYNAMIC_SCALING=true # Disable dynamic scaling and use static worker count
+      - INGEST_DISABLE_DYNAMIC_SCALING=${INGEST_DISABLE_DYNAMIC_SCALING:-true} # Disable dynamic scaling and use static actor count if true(default)
```

Comment:
@drobison00 , In internal benchmarks, we’re noticing a decent performance dip when dynamic scaling is enabled on bo767 on H100:
```
INGEST_DISABLE_DYNAMIC_SCALING=false/default
ingestion_pages_per_sec: 24.4817 
```

```
INGEST_DISABLE_DYNAMIC_SCALING=true 
07-23-25_04: ingestion_pages_per_sec: 30.295
```

Totally see the value of dynamic scaling in resource-constrained settings, but do you think it makes sense to consider disabling it by default to help cases like this?

Are their other parameters, we can tune to make auto scaling less aggressive and ensuring we have at least near the perf we see here ?